.github/workflows/sync_release-manifest.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           # TODO(Rugvip): Remove the create-app dispatch once we've been on the release version for a while
           script: |
             console.log('Dispatching upgrade helper sync - release version');
-            await octokit.actions.createWorkflowDispatch({
+            await github.actions.createWorkflowDispatch({
```

Comment:
I think it's `github.rest.actions`

Reply:
https://github.com/actions/github-script#actionsgithub-script says "pre-authenticated octokit/rest.js client", so wanna try this :grin:

Reply:
https://github.com/actions/github-script#breaking-changes-in-v5 😁